crates/aptos/src/move_tool/mod.rs

```diff
@@ -127,8 +127,6 @@ pub(crate) fn set_bytecode_version(version: Option<u32>) {
     //       environment variables.
     if let Some(ver) = version {
         env::set_var(VAR_BYTECODE_VERSION, ver.to_string());
-    } else if env::var(VAR_BYTECODE_VERSION) == Err(env::VarError::NotPresent) {
-        env::set_var(VAR_BYTECODE_VERSION, "6");
```

Comment:
We probably have to check on the ENV Var area before we get rid of this code entirely.

Reply:
Greg - you added the ability to set the bytecode version via an input to the compiler right? Should we switch to that instead of using this error-prone env var?

Reply:
I can do that in a follow up as agreed during 1:1

Reply:
I added it into the move main branch, but it'll take more work to merge it in.

Just trying to avoid having this bug hanging around, so we won't deploy a new version until the full fix.